File: Exemplos M10/App_IntentDigitalHUB_Ionic/Elgin_IntentDigitalHUB_Ionic-font/src/intentDigitalHubService/BALANCA/Commands/ConfigurarProtocoloComunicacao.ts
import {BalancaCommand} from '../BalancaCommand';

export class ConfigurarProtocoloComunicacao extends BalancaCommand {
  readonly protocoloComunicacao: number;

  constructor(protocoloComunicacao: number) {
    super('ConfigurarProtocoloComunicacao');
    this.protocoloComunicacao = protocoloComunicacao;
  }

  functionParameters(): object {
    return {
      protocoloComunicacao: this.protocoloComunicacao,
    };
  }
}

File: Exemplos M10/App_IntentDigitalHUB_XamarinForms/Elgin_IntentDigitalHUB_XamarinForms-font/E1_Digital_Hub_Xamarin_Forms_M8/IntentServices/Balanca/Commands/ConfigurarProtocoloComunicacao.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Xamarin_Forms_Intent_Digital_Hub.IntentServices.Balanca.Commands
{
    class ConfigurarProtocoloComunicacao : BalanceCommand
    {
        readonly int protocoloComunicacao;

        public ConfigurarProtocoloComunicacao(int protocoloComunicacao) : base("ConfigurarProtocoloComunicacao")
        {
            this.protocoloComunicacao = protocoloComunicacao;
        }
        protected override string FunctionParameters()
        {
            return "\"protocoloComunicacao\"" + ":" + this.protocoloComunicacao;
        }
    }
}
